### PR TITLE
fsdp lazy_init typo

### PR DESCRIPTION
**Pull request by chiaolun**
Minor typo, changed with -> without

### CODE REVIEW

**Review by awgu — APPROVED** [review]
Thanks for catching this! This was my typo 😄

**Review by awgu — CHANGES_REQUESTED** [review]

File: torch/distributed/fsdp/fully_sharded_data_parallel.py [hunk 1]
```diff
@@ -1852,8 +1852,8 @@ def _lazy_init(self) -> None:
         if self._is_root is not None:
             return  # no-op: already initialized
         if not torch.cuda.is_available():
-            # Allow the FSDP constructor to run even with CUDA but check this
-            # once we start real execution
+            # Allow the FSDP constructor to run even without CUDA but check
```
[the remark this answers is not in the document]
Comment (chiaolun) [reply torch/distributed/fsdp/fully_sharded_data_parallel.py]: got it

**Review by awgu — APPROVED** [review]

**Pull request closed without merging**